e2e/tests/oai/001b-infra-metal-lb.sh

```diff
@@ -67,3 +67,4 @@ function _define_ip_address_pool {
 }
 
 _define_ip_address_pool "core" "172.18.16.0/20"
+_define_ip_address_pool "edge" "172.18.48.0/20"
```

Comment:
Curious as to why this is being added now? Trying to understand, Thanks

Reply:
@vjayaramrh It was there in R2 then we removed it because it is not being used. But now we need it for DU because we will use that to communicate with the DU telnet server for bandwidth re-configuration.

Reply:
@arora-sagar Thanks